go.mod

```diff
@@ -1,23 +1,24 @@
 module github.com/github/gh-ost
 
-go 1.21
+go 1.22
```

Comment:
Asked around for a fix, apparently you have to change this to `go 1.22.0`

Reply:
Awesome, thanks @meiji163 🎉